src/test/groovy/edu/ie3/resources/load/BdewLoadProfileTest.groovy

```diff
@@ -30,345 +32,337 @@ class BdewLoadProfileTest extends Specification {
     source = new CsvDataSource(",", resourcePath, new FileNamingStrategy())
   }
 
-  def "The bdew profile g0 should be correct"() {
+  def "The BDEW profile G0 should be correct"() {
     given:
     def data = read(BdewStandardLoadProfile.G0)
 
     when:
-    def suSa = sumValues(data, v -> v.suSa)
-    def suSu = sumValues(data, v -> v.suSu)
-    def suWd = sumValues(data, v -> v.suWd)
+    def keys = ['su', 'tr', 'wi']
+    def results = [:]
```

Comment:
Couldn't this be moved outside the single test, then this only needs to get defined once, right?